PRO/app.js

```diff
@@ -18,7 +18,7 @@ app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 
-// app.use('/migrate', migrateRouter);
+app.use('/migrate', migrateRouter);
```

Comment:
And this as well, keep it commented and only uncomment if you want to migrate database locally.